src/Playground.res

```diff
@@ -891,23 +889,11 @@ module Settings = {
 
     let warnFlagTokens = WarningFlagDescription.Parser.parse(warn_flags)->Result.getOr([])
 
-    let onResetClick = evt => {
-      ReactEvent.Mouse.preventDefault(evt)
-
-      let open_modules = switch readyState.selected.apiVersion {
-      | V1 | V2 | V3 | UnknownVersion(_) => None
-      | V4 | V5 =>
-        readyState.selected.libraries->Array.some(el => el === "@rescript/core")
-          ? Some(["RescriptCore"])
-          : None
-      }
-
-      let defaultConfig = {
-        Api.Config.module_system: "nodejs",
-        warn_flags: "+a-4-9-20-40-41-42-50-61-102-109",
-        ?open_modules,
-      }
-      setConfig(defaultConfig)
+    let onWarningFlagsResetClick = _evt => {
+      setConfig({
+        ...config,
+        warn_flags: "+a-4-9-20-41-50-61-102-109",
```

Comment:
Warnings `40` and `42` no longer exist in the compiler: https://github.com/rescript-lang/rescript/blob/9a8f133f6f1bbc84975efceb8113b767805fbcfd/compiler/ext/warnings.mli#L57-L59

I'll make a separate PR to the compiler repo for updating `Bsc_warnings.defaults_w`, which also still includes these warnings.

Reply:
I've also removed warnings:
- 61: [no longer exists](https://github.com/rescript-lang/rescript/blob/9a8f133f6f1bbc84975efceb8113b767805fbcfd/compiler/ext/warnings.mli#L71-L72)
- 109: [not included in Bsc_warnings.defaults_w](https://github.com/rescript-lang/rescript/blob/9a8f133f6f1bbc84975efceb8113b767805fbcfd/compiler/ext/bsc_warnings.ml#L73)

Reply:
We support older compiler versions here, which might still support those warnings?

Reply:
We should probably have modern (V12-level) warning defaults, just when v12 is selected, and keep this one for the older compilers.

Reply:
The default warnings come from the compiler (specifically, with `RescriptCompilerApi.Compiler.getConfig`) - this string just determines the warnings used when the user clicks on `[reset]` in the 'Warning Flags' section.

Ideally, we'd want to reset the warnings to the initial defaults the compiler returned, but I couldn't figure that out. So I've restored the value of this string back to what it was before (`"+a-4-9-20-40-41-42-50-61-102-109"`)